apps/editing-toolkit/editing-toolkit-plugin/block-patterns/test/class-block-patterns-from-api-test.php

```diff
@@ -41,8 +41,9 @@ public function setUp() {
 			'source_url'    => 'http;//test',
 			'modified_date' => 'dd:mm:YY',
 			'categories'    => array(
-				array(
-					'title' => 'test-category',
+				'test_slug' => array(
```

Comment:
PHP 8 noticed these fields were undefined when going through other tests, which probably means the mock data wasn't fully created in the first place.